client/reducers/info.js

```diff
@@ -24,6 +28,25 @@ export default function infoReducer (state = { pending: true }, action) {
         pending: false,
         error: action.message
       }
+    case GET_FOOD_DETAILS_PENDING:
+      return {
+        ...state,
+        pending: true
+      }
+
+    case GET_FOOD_DETAILS_SUCCESS:
```

Comment:
Hey fun change we can make is you can do this for all cases that will have the same results:

```js
case GET_FOOD_S_PENDING:
case GET_FOOD_DETAILS_PENDING:
      return {
        ...state,
        pending: true
      }
```
Come chat if you need further explanation.